test-result-summary-client/src/TestCompare/TestCompare.jsx

```diff
@@ -68,6 +146,8 @@ export default class TestCompare extends Component {
             </Row>
             <Row>
                 <Col span={24} style={{ textAlign: 'right' }}>
+                    <Checkbox onChange={this.toggleTimeStamp}>TimeStamp</Checkbox>

```

Comment:
Please change the checkbox description to be more clear. 
For example, 
instead of `TimeStamp `, we can use `Remove Timestamp`
instead of `DeepSmith_Match `, we can use `Apply Deep Smith Match`